av1an-core/src/lib.rs

```diff
@@ -232,6 +232,20 @@ pub enum ChunkMethod {
   LSMASH,
 }
 
+#[derive(
+  PartialEq, Eq, Copy, Clone, Serialize, Deserialize, Debug, Display, EnumString, IntoStaticStr,
+)]
+pub enum ChunkOrdering {
+  #[strum(serialize = "long_to_short")]
```

Comment:
This is quite pedantic, but it seems more consistent to serialize these with dashes rather than underscores